src/import/importConfiguration.ts

```diff
@@ -0,0 +1,713 @@
+    meta: parseMeta(yaml.meta, file),
+    implicitRules: yaml.implicitRules,
+    language: parseSimpleCode(yaml.language, 'language', file),
+    text: parseText(yaml.text, file),
+    contained: yaml.contained,
+    extension: yaml.extension,
+    modifierExtension: yaml.modifierExtension,
+    url: yaml.url, // minimum config property
+    version: normalizeToString(yaml.version), // minimum config property
+    name: required(yaml.name, 'name', file),
+    title: yaml.title,
+    status: parseCodeWithRequiredValues(
+      required(yaml.status, 'status', file),
+      ['draft', 'active', 'retired', 'unknown'],
+      'status',
+      file
+    ),
+    experimental: yaml.experimental,
+    date: normalizeToString(yaml.date),
+    publisher: normalizeToArray(yaml.publisher)?.[0]?.name,
+    contact: parseContact(yaml, file),
+    description: yaml.description,
+    useContext: parseUsageContext(yaml.useContext, file),
+    jurisdiction: parseJurisdiction(yaml.jurisdiction, file),
+    copyright: yaml.copyright,
+    packageId: yaml.packageId ?? yaml.id,
+    license: parseSimpleCode(yaml.license, 'license', file),
+    fhirVersion: normalizeToArray(yaml.fhirVersion)?.map(v =>
+      parseSimpleCode(v, 'fhirVersion', file)
+    ), // minimum config property
+    dependencies: parseDependencies(yaml.dependencies),
+    global: parseGlobal(yaml.global),
+    groups: parseGroups(yaml.groups),
+    resources: parseResources(yaml.resources, file),
+    pages: parsePages(yaml.pages, file),
+    parameters: parseParameters(yaml),
+    templates: parseTemplates(yaml.templates, file),
+    template: required(yaml.template, 'template', file),
+    menu: parseMenu(yaml.menu),
+    history: parseHistory(yaml, file)
+  };
+
+  // Remove all undefined variables (mainly helpful for test assertions)
+  removeUndefinedValues(config);
+
+  return config;
+}
+
+function required<T>(value: T, property: string, file: string): T {
+  if (value == null || (Array.isArray(value) && value.length === 0)) {
+    logger.error(`Configuration missing required property: ${property}`, { file });
+  }
+  return value;
+}
+
+function normalizeToString(yamlValue: any): string {
+  if (yamlValue == null) {
+    return;
+  }
+  return `${yamlValue}`;
+}
+
+function normalizeToArray<T>(yamlValue?: T | T[]): T[] | null | undefined {
+  if (yamlValue == null) {
+    return;
+  }
+  return Array.isArray(yamlValue) ? yamlValue : [yamlValue];
+}
+
+function parseSimpleCode(yamlCode: string, property: string, file: string): string {
+  return yamlCode?.startsWith('#') ? parseFshCode(yamlCode, property, file)?.code : yamlCode;
+}
+
+function parseFshCode(yamlCode: string, property: string, file: string): FshCode {
+  // If it has a display, we need to split it out and handle it separately
+  const m = yamlCode.match(/^(.*\S)(\s+"(([^"]|\\")*)")$/);
+  if (m) {
+    const concept = parseCodeLexeme(m[1]);
+    concept.display = m[3].replace('\\"', '"');
+    return concept;
+  }
```

Comment:
This only replaces the first instance of `\"` in the string. If you make the first argument to `replace` into a regular expression, it'll replace each instance.

Reply:
Good catch!